app/star-rating.jsx

```diff
@@ -43,7 +44,7 @@ class Star extends React.Component {
 
      return [
         (<input key={id + "__input"} className="star-rating__input" type="radio" id={id} name={name} value={value} checked={checked} onChange={e => this.onChange(e)}></input>),
-        (<label key={id + "__label"} className="star-rating__label" htmlFor={id}></label>),
+        (<label key={id + "__label"} className={this.props.checked ? "star-rating__clicked" : "star-rating__label"} htmlFor={id}></label>),
```

Comment:
According to BEM convention the name for the clicked state should be `star-rating__label--clicked`. `checked` maybe better than `clicked` in this case because it maps more clearly to the input state.

Reply:
Link to BEM documentation http://getbem.com/introduction/

Reply:
see commit "changes className for checked stars to match BEM convention and reflect aspects of state"